core/src/trezor/lvglui/scrs/app_passkeys.py

```
@@ -0,0 +1,334 @@
+            parent,
+            title,
+            content,
+            font=font_GeistRegular26,
+            bg_color=lv_colors.ONEKEY_BLACK_3,
+        )
+        self.label.set_style_text_color(lv_colors.ONEKEY_WHITE_4, 0)
+        self.label.set_style_text_font(font_GeistRegular30, 0)
+
+
+async def request_credential_details(
+    app_name: str, account_name: str, on_remove
```

Ruff (0.8.2)

Missing type annotation for function argument `on_remove`

(ANN001)